pkgs/by-name/mu/musescore/package.nix

```diff
@@ -77,6 +78,16 @@ stdenv.mkDerivation (finalAttrs: {
 
   dontWrapGApps = true;
 
+  patches = [
+    # Remove musescore > 4.6.3
+    (fetchpatch2 {
+      url = "https://patch-diff.githubusercontent.com/raw/musescore/MuseScore/pull/30667.patch";
+      hash = "sha256-FY2F3UqxXI5tNmKzMWJDxZ7gbXzxDcQzd8TTi2mGvrU=";
+    })
+  ];
+
+  env.NIX_CFLAGS_COMPILE = "-Wno-interference-size";
```

Comment:
I don't like this. I think it is not needed if https://github.com/musescore/MuseScore/pull/30422 is fetched too. My PR also includes the update and it now works:

- https://github.com/NixOS/nixpkgs/pull/454915

I still made it a draft because I wish to fetch the merge commit of https://github.com/musescore/MuseScore/pull/30691 instead of `https://github.com/musescore/MuseScore/pull/30691.patch`.